crates/nargo/tests/test_data/merkle_insert/Prover.toml

```diff
@@ -1,11 +1,11 @@
-old_root = "0x0620374242254671503abf57d13969d41bbae97e59fa97cd7777cd683beb9eb8"
-old_leaf = "0x1cdcf02431ba623767fe389337d011df1048dcc24b98ed81cec97627bab454a0"
+old_root = "0x083b35b32ba24436c1614e4262cb4ad8f98f99cdb5fb0da8932ab2a290034867"
+old_leaf = "0x0b81829b478114d28b964bd382ebff8be0216741aa72ff2896909110aef1704a"
```

Comment:
Is the changed data here expected?

Reply:
Yeah it is. The pedersen changed in the backend and gives different outputs than the old backend. The merkle functions in these tests all use pedersen for compression up to the root.

Reply:
Do we know how regular a change this may be? Seems like this could break a lot of user's code until they update their tomls

Reply:
It seems unlikely that this type of thing would be changing often. Probably only if critical bugs are discovered in the backend. I think there was just a lot of changes (including low-level funcs such as pedersen and schnorr) from the old `barretenberg` we used to reference, to this one that `aztec-connect` is actually based upon. @kevaundray can speak more to this

Reply:
It shouldn't change often(ideally never) because every dependency of the crate would then be broken, for barretenberg this is Noir and the aztec-connect products that are built on it.